src/auro-flight-main.js

```diff
@@ -42,11 +46,55 @@ class AuroFlightMain extends LitElement {
     `;
   }
 
+  connectedCallback() {
+    super.connectedCallback();
+
+    /**
+     * Time template object used by convertTime() method.
+     */
+    this.timeTemplate = {
+      hour: "2-digit",
+      minute: "2-digit",
+      timeZone: this.timeZone
+    };
+
+    this.template = {};
+  }
+
+  /**
+   * @private
+   * @param {string} time - UTC time.
+   * @returns Localized time based from UTC string.
+   */
+  convertTime(time) {
+    const slicedTime = time.slice(0, -6);
+    const newTime = new Date(slicedTime);
+    const localizedTime = newTime.toLocaleString('en-us', this.timeTemplate).replace(/^0+/u, '');
+
+    return localizedTime;
+  }
+
   // function that renders the HTML and CSS into  the scope of the component
   render() {
     return html`
+        <script type="application/ld+json">
+          {
+            "@context": "https://schema.org/",
+            "@type": "Flight",
+            "departureTime": "${this.departureTime}",
+            "arrivalTime": "${this.arrivalTime}",
+            "estimatedFlightDuration": "${this.duration}",
+            "name": "Flight(s) ${this.flights}",
```

Comment:
What is the value of having "Flight(s) hardcoded"?

Reply:
It reads like a partial statement. E.g. `Flight(s) AS55` versus just `AS55`. There is no real spec for `name`, we can do anything we want in there to make use of it. 

I suppose we could replace this with `flightNumber` https://schema.org/flightNumber

Reply:
Agreed, although it would be nice to keep both for the codeshare case -

name: Flights AS2, AA5
flightNumber: AS2, AS4005